src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write code here
+function sortList(list, fatherSelector) {
+  list.sort((a, b) => b.dataset.salary.slice(1).split(',').join('')
+    - a.dataset.salary.slice(1).split(',').join(''));
```

Comment:
I think it's better to write a separate helper function for converting a and b to number

Reply:
i think here it is unnecessary. Only 2 ros of code